scripts/ssv-token-deploy.ts

```diff
@@ -0,0 +1,25 @@
+import { ethers } from 'hardhat';
+
+async function main() {
```

Comment:
mocks are used only for test purposes, they should be moved to test folder.

Reply:
done